src/main/registry.ts

```diff
@@ -18,22 +18,47 @@ if (app) {
   setExternalVBSLocation(vbsDirectory);
 }
 
-const isPeaceInstalled = async () => {
+const isSoftwareInstalled = async (softwareKey: string) => {
   const registryKey =
     'HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall';
   const listResult = await regedit.list([registryKey]);
 
   if (listResult[registryKey].exists) {
     // eslint-disable-next-line no-restricted-syntax
     for (const key of listResult[registryKey].keys) {
-      if (key === 'Peace') {
+      if (key === softwareKey) {
         return true;
       }
     }
   }
   return false;
 };
 
-export default {
-  isPeaceInstalled,
+export const isPeaceInstalled = () => isSoftwareInstalled('Peace');
+export const isEqualizerAPOInstalled = () =>
+  isSoftwareInstalled('EqualizerAPO');
+
+export const getConfigPath = async () => {
+  const isInstalled = await isEqualizerAPOInstalled();
+  if (!isInstalled) {
+    throw new Error('Equalizer APO not installed');
+  }
+
+  try {
+    const registryKey64 = 'HKLM64\\SOFTWARE\\EqualizerAPO';
```

Comment:
Peace checks for Equalizer APO using this registry key first, and then tries the second one if this one fails. From my local testing, this first key tends to always fail but I think it's best to keep both checks regardless, in case someone else installed Equalizer APO at a different location.